### PR TITLE
add lumi.plug.mmeu01 - ZNCZ04LM

### PR DESCRIPTION
**Pull request by starkillerOG**
found from https://github.com/rytilahti/python-miio/discussions/1121#discussioncomment-1250771

### CODE REVIEW

**Comment by starkillerOG** [comment]
@rytilahti could you merge this?

File: miio/gateway/devices/subdevices.yaml [hunk 1]
```diff
@@ -618,6 +618,22 @@
       unit: Watt
       get: get_property_exp
 
+- zigbee_id: lumi.plug.mmeu01
+  model: ZNCZ04LM
+  type_id: -2
```
Comment (rytilahti) [inline miio/gateway/devices/subdevices.yaml:623]: Is it normal for type id to be negative? Otherwise LGTM :+1:

**Review by rytilahti — APPROVED** [review]

File: miio/gateway/devices/subdevices.yaml (hunk 1 above)
Comment (starkillerOG) [reply miio/gateway/devices/subdevices.yaml:623]: No but I don't know the type id and could not find it on the internet, so I set it to a negative value.
Someone who owns the device and a chinese version of the Hub schould get a warning about the device not beeing known and a message that he schould report it here.
Than we can change the -2 to the correct value.

Comment (rytilahti) [reply miio/gateway/devices/subdevices.yaml:623]: Ok :+1:

**Pull request merged**